### PR TITLE
CHANGELOG for model updates published around: Tue Nov 10 19:22:27 UTC 2020

### PR DESCRIPTION
**Pull request by awsmobilesdk**
CHANGELOG for model updates published around: Tue Nov 10 19:22:27 UTC 2020

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-
-- Model updates for the following services
----AWSDynamoDB
 # AWS Mobile SDK for iOS CHANGELOG
 
 ## Unreleased
 
 -Features for next release
 
+## 2.19.1
```
Comment (lawmicha) [inline CHANGELOG.md:8]: doesn't this get populated automatically?

Comment (lawmicha) [inline CHANGELOG.md:8]:
```suggestion
```

Comment (royjit) [reply CHANGELOG.md:8]: Oh does it, I thought it was just for amplify-ios.

Comment (royjit) [reply CHANGELOG.md]: Updated

**Review by lawmicha — APPROVED** [review]

**Pull request merged**